roles/network-multus/tasks/deprovision.yml

```diff
@@ -23,4 +23,20 @@
     dest: /tmp/cni-plugins.yml
 
 - name: Delete cni plugins Resources
-  command: kubectl delete -f /tmp/cni-plugins.yml --ignore-not-found
\ No newline at end of file
+  command: kubectl delete -f /tmp/cni-plugins.yml --ignore-not-found
+
+- name: Render OVS plugin deployment yaml
+  template:
```

Comment:
why don't you use "block:" here, like you do in provision.sh?

Reply:
The block is used to determine whether to run per the relevant boolean. We don't use that logic in deprovision but rather use --ignore-not-found.